### PR TITLE
Add summary for hello method

### PR DESCRIPTION
**Pull request by cevheri**
Test

### CODE REVIEW

File: edabit/01-very-easy/how-edabit-works.py [hunk 1]
```diff
@@ -15,6 +15,9 @@ def hello():
 
 
 def hello():
+    """
+    Summary: This method returns a string
```
Comment (ramazannbilen) [inline edabit/01-very-easy/how-edabit-works.py:19]: test

Comment (ramazannbilen) [inline edabit/01-very-easy/how-edabit-works.py:19]: can i make this sentence as a comment by # mark ?

Comment (ramazannbilen) [reply edabit/01-very-easy/how-edabit-works.py:19]: test

**Review by ramazannbilen — CHANGES_REQUESTED** [review]

File: edabit/01-very-easy/how-edabit-works.py (hunk 1 above)
Comment (ramazannbilen) [inline edabit/01-very-easy/how-edabit-works.py:19]: questions

Comment (cevheri) [reply edabit/01-very-easy/how-edabit-works.py:19]: thanks

**Pull request merged**